lib/manual_relocator.rb

```diff
@@ -134,7 +134,7 @@ def reslug
       sections.each do |section|
         reslug_msg = "Reslugging section '#{section.slug}'"
         new_section_slug = section.slug.gsub(from_slug, to_slug)
-        puts "#{reslug_msg} as '#{section.slug}'"
+        puts "#{reslug_msg} as '#{new_section_slug}'"
```

Comment:
I've added a commit titled "Inline reslug_msg in ManualRelocator#reslug" to do as you've suggested.